test/functional/api/extmark_spec.lua

```diff
@@ -1425,14 +1425,12 @@ describe('API/extmarks', function()
   end)
 
   it('throws consistent error codes', function()
-    local ns_invalid = ns2 + 1
-    eq(
-      "Invalid 'ns_id': 3",
-      pcall_err(set_extmark, ns_invalid, marks[1], positions[1][1], positions[1][2])
-    )
-    eq("Invalid 'ns_id': 3", pcall_err(api.nvim_buf_del_extmark, 0, ns_invalid, marks[1]))
-    eq("Invalid 'ns_id': 3", pcall_err(get_extmarks, ns_invalid, positions[1], positions[2]))
-    eq("Invalid 'ns_id': 3", pcall_err(get_extmark_by_id, ns_invalid, marks[1]))
+    local ns_invalid = ns2 + 1 ---@type integer
+    local err = string.format("Invalid 'ns_id': %d", ns_invalid)
+    eq(err, pcall_err(set_extmark, ns_invalid, marks[1], positions[1][1], positions[1][2]))
+    eq(err, pcall_err(api.nvim_buf_del_extmark, 0, ns_invalid, marks[1]))
+    eq(err, pcall_err(get_extmarks, ns_invalid, positions[1], positions[2]))
+    eq(err, pcall_err(get_extmark_by_id, ns_invalid, marks[1]))
   end)
```

Comment:
Had to update a bunch of tests that were hardcoded to expect namespaces to start at 1 (an assumption that broke when we introduced a namespace into `_defaults.lua`).